ms2query/create_sqlite_database.py

```diff
@@ -0,0 +1,407 @@
+"""
+Functions to create sqlite file that contains 3 tables
+
+The 3 tables are spectrum_data, inchikeys and tanimoto_scores.
+Spectrum_data contains the peaks, intensities and metadata from all spectra.
+Tanimoto_scores contains all tanimoto scores.
+Inchikeys contains the order of inchikeys, that can be used to find
+corresponding indexes in the tanimoto_scores table.
+"""
+
+import sqlite3
+from typing import Dict, List
+import pandas as pd
+import numpy as np
+import time
+from ms2query.app_helpers import load_pickled_file
+from matchms.Spectrum import Spectrum
+import io
```

Comment:
```suggestion
import io
from matchms import Spectrum
from ms2query.app_helpers import load_pickled_file
```

Reply:
Usually the imports from the same package come last. I here also made it a relative import (`.`)

Reply:
I have updated the order in all three files